app/assets/javascripts/app/core/models/discussion.js

```diff
@@ -5,63 +5,59 @@
     .module('council.core')
     .factory('Discussion', Discussion);
 
-  function Discussion(DS, $http, $q) {
+  function Discussion(DiscussionComment, User, DS, $http, $q) {
     var ALLOWED_FIELDS = 'title subtitle body tags'.split(' ');
 
     var Discussion = DS.defineResource({
       name: 'discussion',
       endpoint: 'discussions',
+
       methods: {
         addComment: function(comment) {
-          var refresh = angular.bind(this, this.refresh);
+          var discussion = this;
 
-          return DS.create('discussionComment', comment)
-            .then(refresh);
+          return DS.create('discussionComment', comment, {
+            linkInverse: true,
+            endpoint: 'discussions/' + discussion.id + '/comments'
```

Comment:
The problem is that I can't send the `discussion_id` parameter to the backend, but for it to automatically generate the url it needs to be present, I found this solution on an issue.